docker-compose.qa.yml

```diff
@@ -24,6 +24,8 @@ services:
     environment:
       - BASE_URL
       - BROWSER
+      - USERNAME
```

Comment:
Have you got these used in the seed as we talked about yesterday evening?

Reply:
In my feature branch deployment workflow I have added seeding and I have gone with yet another variable name :)

Reply:
@peterdavidhamilton Happy to change. Let me know what to change to or when ready with your branch.

Reply:
Please let me know what I need to do here if I need to merge your changes into this branch.

Reply:
No I think we should address any tweaks later, just add some secrets.